CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ title: Changelog
 * `<cc-pricing-table>`: add text content to `<cc-button>` elements (toggle State and add buttons) so that they can be identified by assistive technologies.
 * `<cc-zone-input>`: add `tabindex="0"` to the list container so that people can focus and scroll the list without having the change their selection.
 * `resizeObserver`: add `window.requestAnimationFrame` in the resize observer callback to mitigate the `resize observer loop limit exceeded issue`.
+* stories: fix miscellaneous stories not being passed to `enhancedStoryNames`.
```

Comment:
At one point, my testing helpers relied on `story.storyName`. The `storyName` is added by the `enhancedStoryNames` function if i'm right.

Testing would fail because some of our stories were not going through the function so I figured I'd fix them even though we don't rely on these for story testings. 

It might be something that we could check with eslint later on ?

Reply:
* :clap: Well done for the batch of enhanceStoryNames.
* Yes, the `storyName` is optional and added by the `enhancedStoryNames`.
* I'd rather have a different story format that does not require this patch (or even a WDS plugin).